src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -14,7 +14,7 @@ public static class LocalDBTest
         [ConditionalFact(nameof(IsLocalDBEnvironmentSet))]
```

Comment:
Could you remove `SupportsLocalDb` property from the config file and check if `DataTestUtility.LocalDbAppName` has value instead of that?